AFQ/api/group.py

```diff
@@ -916,6 +919,64 @@ def assemble_AFQ_browser(self, output_path=None, metadata=None,
             sublink=page_subtitle_link)
 
 
+class ParallelGroupAFQ():
+    def __init__(self, *args, **kwargs):
+        orig = GroupAFQ(*args, **kwargs)
+
+        if "submitter_params" not in orig.parallel_params:
+            orig.parallel_params["submitter_params"] = {"plugin": "cf"}
+
+        if "cache_dir" not in orig.parallel_params:
+            orig.parallel_params["cache_dir"] = None
+
+        self.parallel_params = orig.parallel_params
+        self.pAFQ_kwargs = [pAFQ.kwargs for pAFQ in orig.pAFQ_list]
+
+    def export_all(self, viz=True, afqbrowser=True, xforms=True, indiv=True):
+        """ Exports all the possible outputs
+
+        Parameters
+        ----------
+        viz : bool
+            Whether to output visualizations. This includes tract profile
+            plots, a figure containing all bundles, and, if using the AFQ
+            segmentation algorithm, individual bundle figures.
+            Default: True
+        afqbrowser : bool
+            Whether to output an AFQ-Browser from this AFQ instance.
+            Default: True
+        xforms : bool
+            Whether to output the reg_template image in subject space and,
+            depending on if it is possible based on the mapping used, to
+            output the b0 in template space.
+            Default: True
+        indiv : bool
+            Whether to output individual bundles in their own files, in
+            addition to the one file containing all bundles. If using
+            the AFQ segmentation algorithm, individual ROIs are also
+            output.
+            Default: True
+        """
+        @pydra.mark.task
+        def export_sub(pAFQ_kwargs, viz, xforms, indiv):
+            pAFQ = ParticipantAFQ(**pAFQ_kwargs)
+            pAFQ.export_all(viz, xforms, indiv)
+
+        # Submit to pydra
+        export_sub_task = export_sub(
+            pAFQ_kwargs=self.pAFQ_kwargs,
+            viz=viz,
+            xforms=xforms,
+            indiv=indiv,
+            cache_dir=self.parallel_params["cache_dir"]
+        ).split("pAFQ_kwargs")
+
+        with pydra.Submitter(
```

Comment:
We should catch this error: https://github.com/nipype/pydra/issues/630 here (so that our users don't see it)